tests/example/CMakeLists.txt

```diff
@@ -14,6 +14,110 @@ function(NEGATIVE_TEST NAME)
     souffle_negative_test(${NAME} example)
 endfunction()
 
+# Make sure that functors are built for the graph_coloring example
```

Comment:
I don't think you need to copy this. Why can you not imitate the functors example.

Reply:
in tests/interface/CMakelist.txt just add 
```souffle_positive_functor_test(graph coloring CATEGORY interface)```